src/code/minimap.asm

```diff
@@ -1,14 +1,15 @@
 Data_002_66F9::
     db   $00, $02, $03, $07, $05, $0A, $0B, $0F, $04, $08, $09, $0E, $06, $0C, $0D, $01
 
-IsMapRoomE8::
+LoadMinimap::
```

Comment:
Oh, so this wasn't a standalone function, it was the leftovers from a patch. Good catch.